Makefile

```diff
@@ -43,7 +43,8 @@ cleanall: clean
 
 .PHONY: test
 test: .build/venv/bin/nosetests test.ini
-	.build/venv/bin/nosetests
+	# All tests are now run with authentication/authorization enabled
```

Comment:
Now that all tests are run with authentication enabled, do we still need the flag `FORCE_AUTHORIZATION_ENABLED`?